node/silkworm/stagedsync/stage_tx_lookup.cpp

```diff
@@ -153,7 +153,7 @@ StageResult unwind_tx_lookup(TransactionManager& txn, const std::filesystem::pat
             while (tx_data && tx_count < body.txn_count) {
                 auto tx_view{db::from_slice(tx_data.value)};
                 auto hash{keccak256(tx_view)};
-                if (lookup_table.seek(mdbx::slice{hash.bytes, kHashLength})) {
+                if (lookup_table.seek(db::to_slice(hash.bytes))) {
                     lookup_table.erase();
                 }
```

Comment:
Since MDBX 0.10.5 this can be simplified as 
```suggestion
                lookup_table.erase(db::to_slice(hash.bytes));
```

Reply:
Done in https://github.com/torquem-ch/silkworm/pull/494/commits/0234a0eee7d2db43a2489a407aa5478b5a798e22